### DISCUSSION TITLE
Feature Request Configure Log Output

### DISCUSSION THREAD

**Discussion by wtmsuperman** [KyryloKuzyk/PrimeTween#61] [Ideas]
I made a script control all tweens by a sequence,but every time build this sequence,there are some error logs.

'useUnscaledTime' was ignored after adding tween/sequence to the Sequence. Parent Sequence controls isPaused/timeScale/useUnscaledTime/useFixedUpdate of all its children tweens and sequences.

I konwn what's going on,actually it's not a error for my senario,and sequence runs correctly.My script set 'useUnscaledTime' default True because most of time it's a UI tween.
These errors are kind of annoying,please give us some options could disable this error.

I could disable all the 'useUnscaledTime' for now,these error are gones.

**Comment by KyryloKuzyk** [comment]
Thanks for reporting this! You get an error because the parent sequence' useUnscaledTime (false) is different from the child tween useUnscaledTime (true). The solution I envisioned for this error message is to use the `Sequence.Create(useUnscaledTime: true)` constructor when creating a sequence and don't set the 'useUnscaledTime' on children tweens. This should also work in your case.

But I agree that this constraint is too strict and nothing bad should happen when sequence.useUnscaledTime == tween. useUnscaledTime. I'll make sure to address that in the next version and log an error only if the 'useUnscaledTime' of the parent sequence is different from the 'useUnscaledTime' parameter of the added tween.

**Comment by KyryloKuzyk** [comment]
@wtmsuperman this issue was fixed in version 1.1.16. https://github.com/KyryloKuzyk/PrimeTween/issues/66
